### ISSUE TITLE
[Enhancement]: Render custom strings

### ISSUE THREAD

**Issue by lennard-brinkhaus** [go-mojito/mojito#45]
### Description

It would be great, to have the possibility to render / show a custom HTML string and not only a path to a specific file.

### Code Snippet

```go
package main

import "github.com/go-mojito/mojito"

func main() {
  mojito.GET("/", func(ctx mojito.RendererContext) {
    ctx.Render(`
      <html>
      <body>
      <h1>Hello World</h1>
      </body>
      </html>
`)
  })
  mojito.ListenAndServe(":8123")
}
```

### Checklist:

- [X] I agree to follow Mojito's [Code of Conduct](../CODE_OF_CONDUCT.md).
- [X] I have checked for existing issues that describe my suggestion prior to opening this one.
- [X] I understand that improperly formatted feature requests reports may be closed without explanation.

**Comment by nilathedragon** [comment]
Thinking about how to design this. I foresee some non-string examples for this.

Let's say a PDF / Word document renderer wouldn't be happy with a string input so `Render(tpl string)` is not viable. I'm thinking either []byte or interface{} but I'd prefer not having interface parameters.

Would there be a situation that is not covered by []byte? Raw file contents and strings both work with that. @0SkillAllLuck

My current proposal would be `Render(tpl []byte) error` and the respective `MustRender(tpl []byte)` resulting in the proposed code snippet changing to:

```go
package main

import "github.com/go-mojito/mojito"

func main() {
  mojito.GET("/", func(ctx mojito.RendererContext) {
    ctx.MustRender([]byte(`
      <html>
      <body>
      <h1>Hello World</h1>
      </body>
      </html>
    `))
  })
  mojito.ListenAndServe(":8123")
}
```

**Comment by 0SkillAllLuck** [comment]
I agree, an alternative would be to have a seperate `ctx.RenderString(tpl string)` function but that would be the worse option I think. So going with your suggestion @nilathedragon looks like the better way to me.
